packages/types/src/parser-options.ts

```diff
@@ -62,13 +61,10 @@ interface ParserOptions {
   extraFileExtensions?: string[];
   filePath?: string;
   jsDocParsingMode?: JSDocParsingMode;
-  loc?: boolean;
   programs?: Program[] | null;
   project?: string[] | string | boolean | null;
-  projectFolderIgnoreList?: (RegExp | string)[];
-  range?: boolean;
+  projectFolderIgnoreList?: string[];
```

Comment:
I haven't verified yet (https://github.com/typescript-eslint/typescript-eslint-examples/issues/9), but it looks to me like we never actually supported the `RegExp` case?

https://github.com/typescript-eslint/typescript-eslint/blob/d8164960d21336d4a726d8179b314f05fe3aeb22/packages/typescript-estree/src/parseSettings/resolveProjectList.ts#L55-L63